terraform/CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing
+
+## Development environment
+
+### Prerequisites
+
+Make sure the following software and tools are installed in the development
+environment.
+
+- `microk8s`
+- `juju`
+- `terraform`
+
+### Prepare Development Environment
```

Comment:
Should we link to a docs page that covers this topic? Options:
- https://discourse.charmhub.io/t/set-up-your-development-environment/7379
- https://juju.is/docs/sdk/dev-setup

Reply:
I checked the links that you provided. It seems they do not exactly match with our  requirements. Thank you for the idea. We can add a discourse page about development environment for Terraform.

Reply:
I created a discourse page:  https://discourse.charmhub.io/t/set-up-your-development-environment-with-microk8s-for-juju-terraform-provider/13109